brave/brave-core
# Add switch to QA Preferences to enable vlog on Android

Add switch to QA Preferences to enable vlog messages for rewards and ads for QA team


Resolves https://github.com/brave/brave-browser/issues/11088

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
1. Connect phone with USB cable to computer, launch Android Device Monitor or Android Studio, enable filtering for Brave browser
2. Install apk
3. Go to kebab => Settings => Developer options => QA Preferences 
4. `Enable vlog for rewards` should be in position off
5. Switch it to position on, restart when prompted 
6. Find these messages in log
```
[VERBOSE1:ads_service_impl.cc(1308)] Migrating ads preferences
[VERBOSE2:ads_service_impl.cc(1334)] Ads preferences are up to date on version 7
```
7. Go to kebab => Settings => Developer options => QA Preferences 
8.  Switch `Enable vlog for rewards` to position off, restart
9. You should not find messages from pt6 in log

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
